coq/coq-syntax.el

```diff
@@ -626,6 +631,8 @@ so for the following reasons:
     ("Import" nil "Import #." t "Import")
     ("Include" nil "Include #." t "Include")
     ("Infix" "inf" "Infix \"#\" := # (at level #) : @{scope}." t "Infix")
+    ("Ltac2 Notation" "lt2not" "Ltac2 Notation # := #." t "Ltac2 Notation")
```

Comment:
and maybe shorten this abbreviation to `lt2n` with a one-letter suffix similarly to lt2p, lt2e…?